.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,10 @@
+name: Deploy to PythonAnywhere
+on:
+  push:
+    branches: [ production ]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          curl -X POST -H "X-funky-password: ${{ secrets.FUNKY_PASSWORD }}" "https://vyxal.pythonanywhere.com/update"
```

Comment:
might I say, whoever wrote this particular bit of code seems like a genius. A great guy, and really handsome too ;)

Reply:
I know right! Golly, I'd give this person a bounty if they specified an answer they wanted it on.

Reply:
(/srs I'll actually give you 50 rep if you give me an answer you want it on)

Reply:
lol ok, let's go with https://codegolf.stackexchange.com/a/237673 just for self-promotion :p